TestgenBookshopAPI/service/scripts/deploy.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# deploy.sh [options] [repository]
+# Deploy the Bookshop from the specified repository.
+# You must already be logged in to the repository.
+# The default repository is the IBM icr.io/integration repository.
+
+cd $(dirname $0)/../deploy
+
+script="$(basename $0)"
+
+USAGE="Usage:
+  ${script} [options]
+  ${script} --tag <tag> [options] [repository]
```

Comment:
Tag is optional (defaults to latest) so the two cases collapse to:
```
${script} [options] [repository]
```

Reply:
Done